src/renderer/components/+network-ingresses/ingresses.scss

```diff
@@ -14,8 +18,13 @@
         display: none;
       }
 
-      span:not(:last-of-type) {
-        margin-right: 1em;
+      > div {
+        overflow: hidden;
+        text-overflow: ellipsis;
+      }
+
+      > div + div {
+        margin-top: 6px;
       }
```

Comment:
These are very easy to break. Add classes.

Reply:
`+` it would be easier to refactor when converting to css-modules